tests/test_input.py

```diff
@@ -12,22 +15,65 @@ def test_input_valid_working_dir_():
     # Assert path to DICOMs exists
     assert valid_filesystem_input.working_directory.is_dir()
 
+    valid_tcia_input = TCIAInput(collection="", patient_ids=[], modalities=[])
```

Comment:
Does this test actually download a data from a collection?

Might need to split these different valid input tests into 3 separate tests, so if one fails it will be easier to work out where it's failing. We can do that in a future PR though.

Reply:
This one doesn't download any data, just confirms that a valid, temporary working directory is created when the constructor is called. We can add more tests that download actual data and make them separate for each one, but these are just sanity check